src/main/java/org/team199/robot2022/commands/RetractClimber.java

```diff
@@ -9,24 +9,23 @@
 import edu.wpi.first.wpilibj2.command.FunctionalCommand;
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
 
-public class RetractClimber extends ParallelCommandGroup {
+public final class RetractClimber extends ParallelCommandGroup {
```

Comment:
```suggestion
public class RetractClimber extends ParallelCommandGroup {
```
Sorry, my bad. The final comment was meant to refer to the member variable not the class. The commands do not have to be final.